src/components/TooltipProvider/TooltipProvider.tsx

```diff
@@ -34,7 +34,9 @@ const DEFAULT_CONTEXT_DATA_WRAPPER: TooltipContextDataWrapper = {
 
 const TooltipContext = createContext<TooltipContextDataWrapper>(DEFAULT_CONTEXT_DATA_WRAPPER)
 
-const TooltipProvider: React.FC<PropsWithChildren> = ({ children }) => {
+interface TooltipProviderProps {}
+
+const TooltipProvider: React.FC<PropsWithChildren<TooltipProviderProps>> = ({ children }) => {
```

Comment:
```suggestion
const TooltipProvider: React.FC<PropsWithChildren<void>> = ({ children }) => {
```

Try doing `void` or `unknown` (preferably `void`) instead and let me know if the compiler or something else complains about it.
The linter doesn't like empty interfaces, so let's try to keep it more concise.